content/articles/working-with-styled-components-in-react/index.md

```diff
@@ -0,0 +1,464 @@
+Import Landing from "./components/Landing" //This is to import the component created in the Landing.js file.
+Import Header from "./components/Header"  // This is to import the component created in the Header.js file
+import './App.css';//Load a set of predefined CSS that will define how HTML elements in the landing page behave.
+ 
+function App() {   //main app
+   return(
+  <div>
+   <Router>
+     <Switch>
+       <Route exact path="/">
+      
+       </Route>
+     </Switch>
+   </Router>
+  </div>)
+
+export default App; 
+
+```
+This component will render a Navbar before styling.
+Now that we have created our components, it is time to route it into the *App.js*. To do so, add the code below between the *Route tags* as shown below:
+
+```JavaScript
+ <Landing/>
+```
+```JavaScript
+ <Header/>
+```
+The final *App.js* should now contain the following code:
+
+```JavaScript
+
+import {BrowserRouter as Router, Switch, Route} from "react-router-dom";
+import Landing from "./components/Landing";
+import Header from "./components/Header";
+import './App.css'
+
+function App() {
+  return(
+  <div>
+   <Router>
+     <Switch>
+       <Route path="/">
+         <Landing/>
+         <Header/>
+       </Route>
+     </Switch>
+   </Router>
+  </div>)
+}
+
+export default App;
+```
+![App.js final](/engineering-education/working-with-styled-components/App.js.png)
+
+### Onto some styling now
+
+Be sure to create an images folder inside the public folder of the application.
+You should have the landing page background image, disney+ icon image, logo-one, and logo-two within the folder.
+You can obtain the images and the icon from [seeklogo](https://seeklogo.com/free-vector-logos/disney?page=4).
+
+
+Let's add some cool features onto our created components- (Landing and Header) and style them using the styled-components depending on what content they hold.
+
+Add the following code between the *Content tags* in the Landing component.
+
+```JavaScript
+          <BgImage/>{/*holder for the landing page back-ground image should be here*/}
+           <CTA>
+               <LogoOne src="images/cta-logo-one.svg" alt='' /> {/*holder for your logo-one should be here, to be styled as imgage*/}
+               <Signup>GET IT ALL HERE</Signup>
+               <Description> {/*holder for a paragraph of text to be styled as p tag*/}
+                   Get premium access to raya and the latest dragon show with a Disney+ subscription. As of 03/04/2021
+                   , the price of Disney+ and the Disney bundle will increase by $1.
+               </Description>
+               <LogoTwo src = "images/cta-logo-two.png" alt='' /> {/*holder for your logo-two should be here, to be styled as image*/}
+            </CTA>
+         
+```
+
+Next, add the code below between the *Nav tags* in the Header component.
+
+```JavaScript
+        <Logo>
+             <img src="/images/logo.svg" alt= "Disney+"/>
+        </Logo> 
+```
+*Bravo if you reached up to this point!*
```

Comment:
Add a screenshot of the expected output at this stage. Do the same for various stages previously that you seem fit.